pkg/pool-stable/contracts/ComposableStablePoolStorage.sol

```diff
@@ -376,6 +376,9 @@ abstract contract ComposableStablePoolStorage is BasePool {
     /**
      * @dev Returns the number of tokens in circulation.
      *
+     * WARNING: in the vast majority of cases this is not a useful value, since it does not include the debt the Pool
```

Comment:
Should we consider not even exposing the "raw" getVirtualSupply? As with the actual supply, they could calculate it externally if they really needed it, and it's not part of the BasePool interface. Would also save a little bytecode.